integration/images/volume-copy-up/Dockerfile

```diff
@@ -12,6 +12,6 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM busybox
+FROM ghcr.io/containerd/busybox:1.32
```

Comment:
When are we building this image? Are we actually building these images as part of our CI? If we do, we should _really_ change that; this looks like an image that will never change.

If we _ARE_ building for every PR, it means; that (looking at [the script that builds these](https://github.com/containerd/containerd/blob/4c7b960cb418d8656c96d424810ef604c97be78d/test/build-test-images.sh#L25-L31);


- We are [authenticating to gcloud](https://github.com/containerd/containerd/blob/4c7b960cb418d8656c96d424810ef604c97be78d/test/build-utils.sh#L29)
- We are installing [`libseccomp2`, `libseccomp2-dev`](https://github.com/containerd/containerd/blob/4c7b960cb418d8656c96d424810ef604c97be78d/test/build-utils.sh#L31-L42)
- enabling buildx and [pulling the `multiarch/qemu-user-static` image to install qemu binfmt](https://github.com/containerd/containerd/blob/4c7b960cb418d8656c96d424810ef604c97be78d/test/init-buildx.sh#L78)
- Building each image for `linux/amd64` and `linux/arm64`
- Push each of them to `gcr.io/k8s-cri-containerd/volume-copy-up:2.0` and `gcr.io/k8s-cri-containerd/volume-ownership:2.0`
- Pull [each of those images as part of the integration tests](https://github.com/containerd/containerd/blob/cf600abecc27200a3c0e1415cd1f6c325eb05edf/integration/common.go#L54-L55)

If would mean we are replacing the image for every CI run, which probably is troublesome (especially with multiple PR's running concurrently).


If we are _NOT_ building these on every PR. We should not change this, as we wouldn't hit rate limits on these, and we don't want to rebuild these images with newer versions of busybox.